VDA5050_EN.md

```diff
@@ -1070,9 +1070,9 @@ released | | boolean | "true" indicates that the edge is part of the base.<br>"f
 Object structure | Unit | Data type | Description
 ---|---|---|---
 **agvPosition** { | | JSON object | Defines the position on a map in world coordinates. Each floor has its own map.
-positionInitialized | | boolean | "true": position is initialized.<br>"false": position is not initialized.
-*localizationScore* | | float64 | Range: [0.0 ... 1.0]<br><br>Describes the quality of the localization and can therefore be used, e.g., by SLAM AGVs to describe how accurate the current position information is.<br><br>0.0: position unknown<br>1.0: position known<br><br>Optional for vehicles that cannot estimate their localization score.<br><br>Only for logging and visualization purposes.
-*deviationRange* | m | float64 | Value for the deviation range of the position in meters.<br><br>Optional for vehicles that cannot estimate their deviation e.g., grid-based localization.<br><br>Only for logging and visualization purposes.
+positionInitialized | | boolean | True: position is initialized. `x`, `y` and `theta` can be trusted.<br>False: position is not initialized. `x`, `y` and `theta` cannot be trusted.<br>Changing to the state to False is only allowed if the vehicle cannot determine its position anymore and the vehicle is not driving (`driving`=False). When entering this state, the vehicle immediately clears the current order. The vehicle should additionally report this state via an error (`errorType` = "FATAL").<br>If `localizationScore` is provided the value should be set to zero.
```

Comment:
Is there a reason for writing True and False? Because part 6.1.4 it is mentioning boolean encoded with "true" and "false".
![image](https://github.com/user-attachments/assets/2b7bbdc2-f0ee-4a19-8fc1-3356af60acf9)

Reply:
No, I did it by accident in the original issue/proposal already.
@collani-bosch please change.
Thanks!